apps/laboratory/src/components/RandomLink.tsx

```diff
@@ -1,6 +1,6 @@
 'use client'
 
-import Link from 'next/link'
+import { Link } from '@chakra-ui/react'
```

Comment:
Used chakra Link component here, since next.js Link component [prefetches](https://nextjs.org/docs/app/api-reference/components/link#prefetch) the pages.